### ISSUE TITLE
broken FBX camera position/orientation

### ISSUE THREAD

**Issue by promethe42** [assimp/assimp#643]
Hello,

using the same Blender file I export a DAE and a FBX.
The DAE loads and renders as expected.
But the FBX camera position is completely broken.

I tried to load the FBX in other programs - such as 3DSMax or Blender itself - and the camera position is fine.
Where could I look in ASSIMP to diagnose the problem further?
Is this a known issue?

https://dl.dropboxusercontent.com/u/39247193/untitled.blend
https://dl.dropboxusercontent.com/u/39247193/untitled.dae
https://dl.dropboxusercontent.com/u/39247193/untitled.fbx

Thank you very much.

**Comment by promethe42** [comment]
So I'm loading both the FBX and the DAE with the exact same (ASSIMP) code, and here is what I get for the camera matrices:

```
main.fbx
transform: mat4(
-100.000000, 0.000004, -0.000039, 0.000000, 
0.000004, 100.000000, 0.000000, 0.000000, 
0.000039, -0.000000, -100.000000, 0.000000, 
1054.672363, 198.692459, 13.696361, 1.000000)

main.dae
transform: mat4(
-0.000000, 1.000000, 0.000000, 0.000000, 
0.000000, 0.000000, 1.000000, 0.000000, 
1.000000, 0.000000, -0.000000, 0.000000, 
10.546721, -0.136964, 1.986925, 1.000000)
```

The last line show there is an obvious difference : the FBX matrix translation is 100x the DAE one.
So you'd think the scale of the FBX matrix would be 0.01, but instead it's 100. So in the end, the FBX matrix has a scale 100 * 100 = 10 000 times bigger than the DAE one.

Where could this come from?

Regards,

**Comment by kimkulling** [comment]
I don't know. Do you have any example?

On Mon, Aug 24, 2015 at 2:01 PM, Jean-Marc Le Roux <notifications@github.com

**Comment by vd3d** [comment]
Is there a fix ? or a way to solve it ?

**Comment by vd3d** [comment]
I have post something on StackOverflow... seems nobody leave here ?

http://stackoverflow.com/questions/35536994/assimp-import-camera

**Comment by kimkulling** [comment]
I am investigating this issue here: https://github.com/assimp/assimp/issues/807

**Comment by kimkulling** [comment]
I guess this is caused by #775 .

**Comment by kimkulling** [comment]
Fixed by  a97a4fb
